File: CPP_Programming/Day_3/program11.cpp
// Description : Write a program to check whether the number is even or odd.

#include<iostream>
using namespace std;

class Check
{
    public :
        int iNo;

    Check(int iNo)
    {
        this->iNo = iNo;
    }

    public : 

        bool CheckEven()
        {
            bool bFlag = false;

            if(iNo % 2 == 0)
            {
                bFlag = true;
            }
    
            return bFlag;
        }

};


int main()
{
    int iValue = 0;
    bool bRet = false;

    cout<<"Enter the number : ";
    cin>>iValue;
    Check cobj(iValue);

    bRet = cobj.CheckEven();

    if(bRet == true)
    {
        cout<<"It is an even number...\n";
    }
    else
    {
        cout<<"It is an odd number...\n";

    }

    return 0;
}

File: Java_Programming/Day_3/program11.java
// Description : Write a program to check whether the number is even or odd.

import java.util.*;

class Check
{
    public int iNo;
    
    Check(int iNo)
    {
        this.iNo = iNo;
    }

    public boolean CheckEven()
    {
        boolean bFlag = false;

        if(iNo % 2 == 0)
        {
            bFlag = true;
        }

        return bFlag;
    }

}

class program11
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number : ");
        int iValue = sobj.nextInt();

        Check cobj = new Check(iValue);

        boolean bRet = cobj.CheckEven();

        if(bRet == true)
        {
            System.out.println("It's an even number...");
        }
        else
        {
            System.out.println("It's an odd number...");
        }

        sobj.close();

    }
}